README.md

```diff
@@ -540,38 +535,52 @@ options:
   -t ENTRY_TYPE, --entry-type ENTRY_TYPE
                         Name of the type (from the CDDL) to interpret the data
                         as.
+  --default-max-qty DEFAULT_MAX_QTY, --dq DEFAULT_MAX_QTY
+                        Default maximum number of repetitions when no maximum
+                        is specified. It is only relevant when handling data
+                        that will be decoded by generated code. If omitted, a
+                        large number will be used.
+  --sanitize            Whether to sanitize/unsanitize CBOR-only values. When
+                        this is enabled, all CBOR data is guaranteed to
+                        convert into YAML/JSON. JSON and YAML do not support
+                        all data types that CBOR/CDDL supports. bytestrings
+                        (BSTR), tags, and maps with non-text keys need special
+                        handling: All strings in JSON/YAML are text strings.
+                        If a BSTR is needed, use a dict with a single entry,
+                        with "zcbor_bstr" as the key, and the byte string (as
```

Comment:
This starts to sound complicated, should we have a sample/test which showcases this and point to that from here instead of verbatim explaining the structure needed for replicating some cbor primitive?

Reply:
Added a test case/example.